python/pyarrow/_fs.pyx

```diff
@@ -521,6 +721,28 @@ cdef class FileSystem(_Weakrefable):
             The path of the file or the directory to be moved.
         dest : str
             The destination path where the file or directory is moved to.
+
+        Examples
+        --------
+        Create a new folder with a file:
+
+        >>> local.create_dir('/tmp/other_dir')
```

Comment:
I didn't want to delete the file from `tmp_path` so I created a new one in `tmp` for `move` and `delete` examples and then deleted the newly created files/folders.